### PR TITLE
added chassis method for align robot facing to setpoint

### PR DESCRIPTION
**Pull request by outsidermm**

### CODE REVIEW

**Comment by james-ward** [comment]
I think the controller needs to be persistent - it should be a member variable in the `Chassis` class, then it gets called each `engage()` loop on the chassis (with a check to see if the heading hold is active or not)

**Review by james-ward — APPROVED** [review]
Good work

File: .deploy_cfg
Comment (auscompgeek) [inline .deploy_cfg:1]: :thinking: why was this added? RobotPy no longer uses this file
